__tests__/app.test.js

```diff
@@ -12,20 +13,32 @@ afterAll(() => {
 });
 
 
-  describe("/api/anything", () => {
+describe("/api", ()=>{
+    test("Get: 200 - responds with all the endpoints", ()=>{
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            expect(typeof body.endpoints).toBe("object");
+            expect(body.endpoints).toEqual(endpoints);
```

Comment:
🌟 🥳 
I think the test on line 22 is not entirely necessary, as this test will fail if body.endpoints is not an object. If you prefer to have it, that's fine too 😄

Reply:
I will leave this one for now, I am having a question.
Should I create a new branch specifc to changes or can do this in my current branch and commit normally, having this change once i make the final push ?